modules/monitoring/templates/ssl.conf.erb

```diff
@@ -33,7 +33,7 @@ apply Service "<%= property['url'] %> - reverse DNS" {
   check_command = "reverse_dns"
   check_interval = 7m
   vars.hostname = "<%= property['url'] %>"
-  vars.regex = "^cp[0-9]+\\.miraheze\\.org$$"
+  vars.regex = "^(test[0-9]+|cp[0-9]+)\.miraheze\.org$$"
```

Comment:
Should be `^(test|cp)[0-9]+\\.miraheze\\.org$$` or similar. Needs to match `(test|cp)[0-9]+\.miraheze\.org` exactly I believe.